src/content/blog/new-domain-new-discord-same-project.md

```diff
@@ -0,0 +1,60 @@
+will be the place for you to chat and receive news about StudioCMS from now on! *Don't worry, the blog isn't going anywhere ;)*
+
+Moving to the new Discord Server has already allowed us to do some pretty cool things. For example, yours truly has been hard at work at our new Open 
+Source Discord Bot, [Apollo](https://github.com/withstudiocms/apollo)! If you're on the Astro Discord, you'll know what they're good at. PTAL notifications, 
+moderation tools (soon™️), support forum utilities, it's all there.
+
+You can even use this Bot in your own Servers! All you need is a GitHub App, Discord App, and a libSQL database connection, for which we obviously use 
+and highly recommend [Turso](https://turso.tech/). You can take a look at the [README file](https://github.com/withstudiocms/apollo?tab=readme-ov-file#apollo) 
+to learn more.
+
+But if you really **did** read the title (or clicked one of the links in this article), you might've also noticed another thing. Thus, it is with great 
+pleasure that I announce... *Drumroll please...* 🥁
+
+## The new StudioCMS GitHub Org
+That's right, along with the Discord Server we've also got [our own GitHub Organization](https://github.com/withstudiocms) now! 
+Similar to the Discord, we've felt that only having access to the repository without any organization-management options was holding us back a tiny bit, 
+so here's to [github.com/withstudiocms](https://github.com/withstudiocms).
+
+This new Org allows us to have better team management, de-clutter our Monorepo and iterate faster on any changes we want to implement. It's small things, 
+really - for example, our website now has its own repository. It's not much, but it's definitely better than having to navigate an entire monorepo to update a blog post. 
+*Speaking from experience here.*
+
+Okay, we've got one last change. Take a look at your browser's address bar. Do you see that? Yeah, that's right. **I introduce to you:**
+
+## studiocms.dev
+That's right, I just realized I said "That's right" below every heading. Terribly sorry for that.
+
+In other news, we've moved to a new domain! `studiocms.dev` is going to be our new home starting from now on. According to ChatGPT, it's supposed to give us a 
+"more professional look", however I just think it sounds nicer. Who uses .xyz domains anyway, .dev is the new black.
+
+We want to thank Astrolicious again for giving our small project a kickstart back when it was still in its infancy. None of this would have been possible without 
+them. We're going to keep chatting along in both the Astro and Astrolicious servers, and hope you'll join us on our [new Discord Server](https://chat.studiocms.dev). 
+
+We've got some exciting stuff coming up, and we're going to be in touch soon about the coveted `beta.8` that's been haunting mine and Adam's dreams.
+
+Thanks and until next time, <br />
+**Louis and the StudioCMS Team**
```

Comment:
```suggestion
**Louis and the StudioCMS team**
```

Reply:
I refer to the above, Chicago Manual of Style